src/services/index.ts

```diff
@@ -824,14 +826,14 @@ export default class Provider extends CloudGraph.Client {
           result.entities[existingServiceIdx] = {
             className: serviceClass.constructor.name,
             name: serviceData.name,
-            mutation: serviceClass.mutation,
+            mutation: generateEntityMutations(schemaName),
```

Comment:
So doe this change `mutation` to be an object `{ add: ...., delete: ... }`?

Reply:
And if so, can we delete the mutation files?

Reply:
And if so, can we delete the mutation files?

Reply:
Yes, that's the case. Those files won't be required anymore. I didn't know if include that as part of this PR, but I can delete them if you want.

Reply:
Lets make sure this strategy works first so we can easily rollback if not but ok sounds good!